Makefile

```diff
@@ -20,6 +20,27 @@ update-db-schema:
 caddy:
 	caddy run --watch --config ee/tabby-webserver/development/Caddyfile
 
+MODEL ?= ""
+CHAT_MODEL ?= ""
```

Comment:
prefer setting them inside of `~/.tabby/config.toml` instead of in every invoking